cmd/notation/sign.go

```diff
@@ -35,6 +35,24 @@ func signCommand(opts *signOpts) *cobra.Command {
 	command := &cobra.Command{
 		Use:   "sign [reference]",
 		Short: "Signs artifacts",
+		Long: `
+Prerequiste: a signing key needs to be configured using the command "notation key".
```

Comment:
Could you add a title for the help command? Otherwise, it will show

```console
$ ./notation sign --help

Prerequiste: a signing key needs to be configured using the command "notation key".

Example - Sign a container image using the default signing key, with the default JWS envelope:
  notation sign <registry>/<repository>:<tag>

...
```

Reply:
@shizhMSFT  Good catch. Fixed. 

Just be aware of the value in the `Short` field will be omitted when defines a `Long` field.

The current CLI help UX for `notation sign`:

```
 ./notation sign --help

Sign OCI artifacts

Prerequisite: a signing key needs to be configured using the command "notation key".

Example - Sign a container image using the default signing key, with the default JWS envelope:
  notation sign <registry>/<repository>:<tag>

Example - Sign a container image using the default signing key, with the COSE envelope:
  notation sign --envelope-type cose <registry>/<repository>:<tag>

Example - Sign a container image using the specified key name
  notation sign --key <key_name> <registry>/<repository>:<tag>
```